OnlineGameStore.Application/Models/Requests/GameRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace OnlineGameStore.Application.Models.Requests
+{
+    public class GameRequest
+    {
+        public string? Name { get; set; }
+
+        public string? Description { get; set; }
+
```

Comment:
I think you accidentally removed the `key` property :\

Reply:
Yes, thnx, a added key back

Reply:
Key must be string